CHANGELOG.rst

```diff
@@ -4,10 +4,18 @@ Changelog
 3.0.0b12 (unreleased)
 +++++++++++++++++++++
 
+Features:
+
+- The behavior to apply when encountering unknown fields while deserializing can be controlled with the ``unknown`` option (:issue:`524`).
+  It makes it possible to either "drop", "pass" or "raise".
```

Comment:
This needs to be updated to "exclude", "include", or "raise".

Reply:
Done